Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -91,6 +91,12 @@ theorem eq_one_iff {N : Subgroup G} [N.Normal] (x : G) : (x : G ⧸ N) = 1 ↔ x
   refine QuotientGroup.eq.trans ?_
   rw [mul_one, Subgroup.inv_mem_iff]
 
+/-- Note: `range_mk'` is a lemma about the primed constructor `QuotientGroup.mk'`, not a
+  modified version of some `range_mk` -/
```

Comment:
I think should be an ordinary comment starting with `--` rather than a docstring, since this comment is not really about the content of the theorem.
```suggestion
-- Note: `range_mk'` is a lemma about the primed constructor `QuotientGroup.mk'`, not a
--  modified version of some `range_mk`
```